### ISSUE TITLE
Propose to add Redfish Event validation against Event schema

### ISSUE THREAD

**Issue by lindappwu** [DMTF/Redfish-Event-Listener#31]
Event listener currently doesn't have the capability to flag errors for below conditions: 
1. Event data is not compliant to the Event schema
2. Numbers of MessageArgs is not aligned with the numbers specified in the Message Registry 

Does it sound like a good idea to add these checking's to the event listener?

Welcome other thoughts as well, thanks.

**Comment by jautor** [comment]
I agree this would be a useful "mode" to add to the tool as an enhancement.  No need for a separate tool for this.  

To build on @lindappwu points:
- Point to a message registry repository for standard plus OEM registries.
- Validate `MessageId` format, and existence of an Id within the specified message registry (check minor version number, too)
- Perhaps keep track of received `EventId` values to ensure they are unique per service at least while listener is running
